samples/Http2SampleApp/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM microsoft/dotnet:2.0.0-runtime-stretch
```

Comment:
Out of curiosity when would I choose this over `microsoft/aspnetcore:2.0.0-stretch` or vice versa?

Reply:
I suppose it's OK. That will bring in the runtime store, but I think that's probably fine.